changelog/unreleased/issue-1542

```diff
@@ -0,0 +1,9 @@
+Enhancement: Add --dry-run/-n option to backup command.
+
+We added a new --dry-run/-n option to backup, which performs all the normal
+steps of a backup without actually writing data. Passing -vv will log
+information about files that would be added, allowing fast verification of
+backup options without any unnecessary write activity.
```

Comment:
I suggest changing the text to the following:

~~~
We added a new --dry-run/-n option to backup the backup command,
which performs all the normal steps of a backup without actually writing
any changes to the repository. Passing -vv will log information about
files that would be added, allowing fast verification of backup options
without any unnecessary write or upload activity.
~~~